x-pack/plugins/apm/public/components/app/ErrorGroupOverview/index.tsx

```diff
@@ -57,7 +54,7 @@ const ErrorGroupOverview: React.SFC<ErrorGroupOverviewProps> = ({
       <ErrorGroupOverviewRequest
         urlParams={urlParams}
         render={({ data }) => (
-          <List urlParams={urlParams} items={data} location={location} />
+          <ErrorGroupList urlParams={urlParams} items={data} />
```

Comment:
Should `location` not be passed to `ErrorGroupList`?

Reply:
Yup, and it needs to be re-passed here from the service tabs. Both fixed.